### PR TITLE
Add select revote

### PR DESCRIPTION
**Pull request by NerdsvilleCEO**

### CODE REVIEW

File: fedora_elections/elections.py [hunk 1]
```diff
@@ -210,20 +208,27 @@ def vote_select(election):
...
-                        value=int(candidate.data),
-                    )
-                    SESSION.add(new_vote)
-                SESSION.commit()
-
+                candidates = [
+                              candidate
+                              for candidate in form
+                              if candidate and candidate.short_name not in ['csrf_token', 'action']
+                             ]
```
Comment (pypingou) [inline fedora_elections/elections.py]: Indentation here seems buggy, way too large compared to the rest of the file

**Comment by NerdsvilleCEO** [comment]
The indentation should be fixed :)

**Comment by NerdsvilleCEO** [comment]
@pypingou just checking if there are any updated :D

**Comment by pypingou** [comment]
Sorry I'm currently AFK. I still want to test it before merging it but I know it looks good :)

**Comment by pypingou** [comment]
This seems to no longer apply, could you rebase it?

Thanks

**Comment by NerdsvilleCEO** [comment]
rebasing now one sec

**Comment by pypingou** [comment]
This seems odd, did you merge or rebase?

**Comment by NerdsvilleCEO** [comment]
oh, I merged it, I thought that's what it meant... not sure how to rebase

**Comment by pypingou** [comment]
pull the latest changes from develop, go to your branch and ``git rebase develop`` :)

**Comment by NerdsvilleCEO** [comment]
Did I do it right, that was scary I had no clue what I was doing XD

**Comment by pypingou** [comment]
The merge commit still surprises me, but otherwise it looks correct :)

**Comment by NerdsvilleCEO** [comment]
Sorry about that, I am used to merging, never knew about rebase

**Comment by pypingou** [comment]
Could you try once again?

Sorry for all this, it seems I managed to mess-up my commits between master and develop and between fedorahosted and github... What a mess... :)

**Comment by NerdsvilleCEO** [comment]
Ok, sorry about this once again

**Comment by pypingou** [comment]
It should go something like this
````
git checkout develop
git pull --rebase <the origin for this repo> develop
git checkout add-select-revote
git rebase develop
<fix conflicts>
git push -f # -f since you re-wrote history
````

**Comment by NerdsvilleCEO** [comment]
That's exactly what I did with the rebase last time, except I committed like git push origin +add-select-revote, I bet the merge was left over from before

**Comment by NerdsvilleCEO** [comment]
Oh I see what I didn't do, I merged instead of rebase pull

**Comment by NerdsvilleCEO** [comment]
Is it right this time

**Comment by pypingou** [comment]
Perfect! Thanks :)

**Comment by NerdsvilleCEO** [comment]
No problem :) I learned something new today :+1: Thanks!

**Comment by pypingou** [comment]
I'm going to merge this PR as is

- One of the reasons the tests are failing is that we check in the UI to hide the 'revote' button if the election type does not support revote, but we do not check for it in the code, so calling the URLs directly doesn't fail as it should :)

**Comment by NerdsvilleCEO** [comment]
Ok, let me finish up the other types today and it should fix the tests then! :) give me an hour and I should have those finished

**Pull request merged**